misc/version.sh

```diff
@@ -26,9 +28,23 @@ if test -z "${GIT_VERSION}" -a -n "${FILE_VERSION}"; then
     exit 0
 fi
 
+DEPS=""
+if test -f ${SRCDIR}/check-deps/have_libdw; then
+    DEPS="${DEPS} dwarf"
+fi
+if test -f ${SRCDIR}/check-deps/have_libpython2.7; then
+    DEPS="${DEPS} python"
+fi
+if test -f ${SRCDIR}/check-deps/have_libncurses; then
+    DEPS="${DEPS} tui"
+fi
+if [ "x${DEPS}" != "x" ]; then
+    DEPS="(${DEPS} )"
+fi
+
 if test -z "${FILE_VERSION}" -o "${CURR_VERSION}" != "${FILE_VERSION}"; then
     # update file version only if it's different
-    echo "#define UFTRACE_VERSION  \"${CURR_VERSION}\"" > ${VERSION_FILE}
+    echo "#define UFTRACE_VERSION  \"${CURR_VERSION} ${DEPS}\"" > ${VERSION_FILE}
```

Comment:
Nice work!  This way the output of uftrace info will also be changed.

I have a little suggestion.  It'd be nice to have no space if no dependency is detected.  So how about this?

```
if [ "x${DEPS} != "x" ]; then
  DEPS=" (${DEPS} )"
fi
...
  echo "#define UFTRACE_VERSION  \"${CURR_VERSION}${DEPS}\"" > ${VERSION_FILE}
```

Reply:
Sure.